### PR TITLE
Explaining what adding a platform does.

### PR DESCRIPTION
**Pull request by gewenyu99**
During office hours, a hacker was confused about what adding platforms does. Explaining to them that adding web platforms prevents unwanted cross-domain access helped answer his question and reassure him about security concerns.

To him, a security-conscious individual, this was an important piece of missing information.

This PR aims to add a missing explanation.

### CODE REVIEW

**Comment by gewenyu99** [comment]
@abnegate @lohanidamodar I have a question for you guys.

What does adding a platform do for mobile/flutter apps? Is it a similar reason to CORS protection? How should I explain what they do to an Appwrite developer?

**Comment by lohanidamodar** [comment]
@gewenyu99 We use platforms to validate request from mobile SDKs, without adding platforms for mobile applications Appwrite throws error and suggests to add platform.

File: app/views/docs/getting-started-for-web.phtml [hunk 1]
```diff
@@ -27,7 +27,7 @@ $demos = $platform['demos'] ?? [];
 
 <p>To init your SDK and interact with Appwrite services, you need to add a web platform to your project. To add a new platform, go to your Appwrite console, choose the project you created in the step before and click the 'Add Platform' button.</p>
 
-<p>From the options, choose to add a <b>web</b> platform and add your client app hostname. By adding your hostname to your project platform, you are allowing cross-domain communication between your project and the Appwrite API.</p>
+<p>From the options, choose to add a <b>web</b> platform and add your client app hostname. By adding your hostname to your project platform, you are allowing cross-domain communication between your project and the Appwrite API. Only web apps hosted on domains specified in a web platform will be able to make requests to your Appwrite instance, preventing unwanted access from malicious actors.</p>
```
Comment (eldadfux) [inline app/views/docs/getting-started-for-web.phtml:30]: Can we get this ready for 0.14, and also mention something about wildcard hostnames?

**Comment by eldadfux** [comment]
> @gewenyu99 We use platforms to validate request from mobile SDKs, without adding platforms for mobile applications Appwrite throws error and suggests to add platform.

To clarify further, this protection is only in the Appwrite SDK level and can be bypassed by passing the correct headers. The ability to distinguish platforms like that gives us the possibility for better stats reporting in the future.

File: app/views/docs/getting-started-for-web.phtml (hunk 1 above)
Comment (gewenyu99) [reply app/views/docs/getting-started-for-web.phtml:30]: Goooood point! I'll deal with that tomorrow :D

**Comment by gewenyu99** [comment]
Screen Shot 2022-05-05 at 4 58 56 PM
Updated description to mention wildcards. Do you guys feel there's too much text here?

**Comment by gewenyu99** [comment]
Screen Shot 2022-05-12 at 1 29 08 PM
@TorstenDittmann Fixed the orphaned punctuation :)

**Pull request merged**